docs/knn/index.md

```diff
@@ -125,6 +125,34 @@ GET my-knn-index-1/_search
 }
 ```
 
+The formulas we use to perform calculations are as follows:
+
+Euclidian Distance:
+<p>
+
+\[ Distance(X, Y) = \sqrt{\sum_{i=1}^n (X_i - Y_i)^2} \]
+
+</p>
+
+Cosine similarity:
+<p>
+  \[ {A &middot; B \over \|A\| &middot; \|B\|} =
+
+  {\sum_{i=1}^n (A_i &middot; B_i) \over \sqrt{\sum_{i=1}^n A_i^2} &middot; \sqrt{\sum_{i=1}^n B_i^2}}\]
+ where \(\|A\|\) and \(\|B\|\) represent normalized vectors.
+
+</p>
+
+After calculations, we perform the following conversions to get the final Elasticsearch score. Note that these conversions are how we calculate approximate k-NN scores. For custom scoring conversions, see the [custom scoring](#Pre-filtering-with-custom-scoring) section.
+
+[Euclidian Distance](https://github.com/opendistro-for-elasticsearch/k-NN/blob/0da03b29f1367b7f555e14b4ea4002626160bb35/src/main/java/com/amazon/opendistroforelasticsearch/knn/index/KNNWeight.java#L113):
+```
+Elasticsearch score = 1/(1 + Euclidian distance)
```

Comment:
I'd add a couple spaces here for readability and consistency, regardless of how it looks in code. `1 / (1 + Euclidian distance)` I think this pattern occurs four times.

Reply:
I found three instances of this, and I've added spaces for readability, but please let me know if I need to edit anywhere else.